docs/sql-ref-syntax-ddl-drop-table.md

```diff
@@ -26,6 +26,8 @@ if the table is not `EXTERNAL` table. If the table is not present it throws an e
 
 In case of an external table, only the associated metadata information is removed from the metastore database.
 
+If the table is cached, the command uncaches the table and all its dependents such as views. Thereby the dependents should be cached again explicitly.
```

Comment:
let's remove `such as views`. Permanent view is not dependents even if it refers to the table.

Reply:
The sentence "Thereby the dependents should be cached again explicitly." is not needed too, I believe.